Docs: fix textobjects

kuator opened the pull request:
Small fixes

stsewd commented on README.md:
```diff
@@ -211,6 +211,7 @@ similar to `ip` (inner paragraph) and `ap` (a paragraph).
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
   textobjects = {
+    enable = true,
```
This isn't needed. They stopped working because the module was moved to a submodule. See the examples in the docs for the new settings.

stsewd commented on README.md:
```diff
@@ -267,7 +269,8 @@ or previous function.
 require'nvim-treesitter.configs'.setup {
   textobjects = {
-    move {
+    enable = true,
+    move = {
```
```suggestion
    move = {
```

stsewd replied:
This also needs to be fixed in https://github.com/nvim-treesitter/nvim-treesitter/blob/7f81df44098a7fef93d6901a06b8312c5b8619d7/doc/nvim-treesitter.txt#L365-L365

kyazdani42 commented on README.md:
```diff
@@ -243,6 +244,7 @@ like function parameters or arguments.
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
   textobjects = {
+    enable = true, 
```
This isn't needed, the textobject itself is not a module, it has submodules that needs to be activated one by one.

kuator commented:
I updated nvim-treesitter, and textobjects stopped working for me. I updated configuration with code from readme, no effect. So I added `enabled=true` manually , and textobjects module started working. But I see it should work as is.

kuator commented:
> I updated nvim-treesitter, and textobjects stopped working for me. I updated configuration with code from readme, no effect. So I added `enabled=true` manually , and textobjects module started working. But I see it should work as is.

I removed `enable=true` from my config and it works. My bad :(
